proto/raft_cmdpb.proto

```diff
@@ -109,25 +109,35 @@ message CompactLogResponse {
     
 }
 
+message TransferLeaderRequest {
+    optional metapb.Peer peer =	1;
```

Comment:
it seems a bug of github...
open the 'view' button and view the whole file, you'll see there is actually a space here.